datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -229,7 +229,7 @@ CREATE TABLE dcsa_im_v3_0.service (
     carrier_service_code varchar(5),
     carrier_service_name varchar(50),
     tradelane_id varchar(8) NULL REFERENCES dcsa_im_v3_0.tradelane(id),
-    universal_service_reference varchar(8) NULL
+    universal_service_reference varchar(8) NULL UNIQUE
```

Comment:
The USR **cannot** be unique the service table.

(Two carrier services can use the same `USR`).

Reply:
Related I remember that there has been confusion on this topic before, so I think we should add a comment to the SQL explaining why we do not have a `UNIQUE` here.